plugins/rtmp-services/data/services.json

```diff
@@ -1271,7 +1271,11 @@
                 }
             ],
             "recommended": {
-                "max video bitrate": 5000
+                "keyint": 4,
+                "profile": "main",
+                "max video bitrate": 3000,
+                "max audio bitrate": 256,    
+                "x264opts": "tune=zerolatency threads=8"
```

Comment:
You absolutely should **not** set the threads parameter, that is up to the user's computer and you cannot set a generic value and expect it to work reliably.

Also the zerolatency tune and main profile both limit the quality unnecessarily. Piczel is using HLS with a long segment duration + high keyframe interval, so the latency reduction gained from this is outweighed massively by the latency of the rest of the stack, and pretty much pointless.

Reply:
I see you're also using WebRTC now, in that case `zerolatency` makes a *little* more sense. However setting main profile is probably still not a good idea. And setting the threads is *absolutely* not a good idea.

Reply:
Our vendor for our WebRTC server suggests both main and setting threads to 8 (as apparently chrome doesn't handle playback well with more than 8 threads, supposedly? 

Please don't speculate on our stream server backend though, we're quite familiar with HLS at least, even if webrtc is new and scary.

I'll get some answers and test your suggestions and update the PR

Reply:
While I'm not too familiar with sliced threads I don't think it changes the decoder complexity. So whatever decoder Chrome is using shouldn't care whether you use 1 or 16 threads to encode (just like with regular x264 encoding).

However sliced threads do not scale well in terms of speed and quality (the quality loss with more threads is more severe than with x264's regular threading model), so setting a maximum makes sense. x264 itself limits sliced threads more harshly than normal threads, but I think it would still allow but to 16 or so for 1080p content. Unfortunately there is no option for maximum threads in x264, and the problem with just setting 8 is that users with 4c/4t CPUs (which is still a lot) would likely see worse performance, and produce worse quality than with the defaults. So in general I'm not comfortable with having a more-or-less hardcoded thread count in the services file.